shared/modules/types.go

```diff
@@ -15,6 +15,8 @@ type GenesisState struct {
 type BaseConfig struct {
 	RootDirectory string `json:"root_directory"`
 	PrivateKey    string `json:"private_key"` // TODO (pocket/issues/150) better architecture for key management (keybase, keyfiles, etc.)
+	ConfigPath    string `json:"config_path"`
```

Comment:
These fields that are introduced are not used anywhere. What's the reasoning?

Also, we have a very similar structure under `runtime` package. What's the plan there?

Reply:
Refactored mostly in https://github.com/pokt-network/pocket/pull/235/commits/1924e0e7e3c6988d13cb0b4a2a4edc5c07c64f2b

Relying more on abstractions